modules/apigee_edge_teams/src/CompanyMembershipObjectCache.php

```diff
@@ -98,14 +98,20 @@ public function __construct(CacheFactoryInterface $cache_factory, MemoryCacheFac
    * {@inheritdoc}
    */
   public function saveMembership(string $company, CompanyMembership $membership): void {
+    $expiration = $this->persistentCacheExpiration;
+
+    // Do not save in cache, if expiration is set to 0.
+    if (0 === $expiration) {
```

Comment:
@phdhiren What about other places? (e.g: app caches) Why just here? Configurable cache expiration is a fundemantal features of Apigee Edge module.

Reply:
Thanks for the input @mxr576 
Let me add separate ticket to check this across Apigee Edge.